containers/proxy-tftpd-image/tftp_wrapper.py

```diff
@@ -82,158 +97,229 @@ def contentFilter(self):
 
     def get_handler(self, server_addr, peer, path, options):
         return TFTPHandler(
-            server_addr, peer, path, options, self._root, self._httpHost, self._capath
+            server_addr,
+            peer,
+            path,
+            options,
+            self._root,
+            self._httpHost,
+            self._proxyFqdn,
+            self._capath,
         )
 
+
 def get_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--ip", type=str, default="0.0.0.0", help="IP address to bind to")
+    parser.add_argument(
+        "--ip", type=str, default="0.0.0.0", help="IP address to bind to"
+    )
     parser.add_argument("--port", type=int, default=69, help="port to bind to")
     parser.add_argument(
         "--retries", type=int, default=5, help="Number of per-packet retries"
     )
     parser.add_argument(
-        "--timeout", type=int, default=2, help="Timeout for packet retransmission in seconds"
+        "--timeout",
+        type=int,
+        default=2,
+        help="Timeout for packet retransmission in seconds",
+    )
+    parser.add_argument(
+        "--root",
+        type=str,
+        default="/srv/tftpboot",
+        help="Root of the static filesystem",
     )
     parser.add_argument(
-        "--root", type=str, default="/srv/tftpboot", help="Root of the static filesystem"
+        "--httpHost",
+        type=str,
+        default="localhost",
+        help="Hostname where to forward HTTP requests",
     )
     parser.add_argument(
-        "--httpHost", type=str, default="localhost", help="Hostname where to forward HTTP requests"
+        "--proxyFqdn",
+        type=str,
+        help="Hostname of the proxy for retail filtering",
+        default="localhost",
     )
     parser.add_argument(
-        "--logLevel", choices=['error', 'warning', 'info', 'debug'], default="debug", help="Logging level for the server"
+        "--logLevel",
+        choices=["error", "warning", "info", "debug"],
+        default="info",
+        help="Logging level for the server",
     )
     parser.add_argument(
-        "--configFile", type=str, default="/etc/uyuni/config.yaml", help="Path to configuration file"
+        "--configFile",
+        type=str,
+        default="/etc/uyuni/config.yaml",
+        help="Path to configuration file",
     )
     parser.add_argument(
-        "--caPath", type=str, default="/usr/share/uyuni/ca.crt", help="Path to CA certificate"
+        "--caPath",
+        type=str,
+        default="/usr/share/uyuni/ca.crt",
+        help="Path to CA certificate",
     )
     return parser.parse_args()
 
+
 def main():
     args = get_arguments()
     logging.getLogger().setLevel(logging.getLevelName(args.logLevel.upper()))
 
     try:
         with open(args.configFile) as source:
             config = yaml.safe_load(source)
-            args.httpHost = config['proxy_fqdn']
+            args.proxyFqdn = config.get("proxy_fqdn", "localhost")
```

Comment:
Do we get the `proxy_fqdn` config from the container config?

Reply:
Yes, in our deployment it's always set correctly from the config file. But I see a mistake here. As a `get` default there should be `args.proxyFqdn`.
This is more for standalone usage without using any config file.